src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -126,4 +126,55 @@ public static string Format(string resourceFormat, object p1, object p2, object
             return string.Format(resourceFormat, p1, p2, p3);
         }
     }
+
+    internal partial class StringsHelper
```

Comment:
NIT: Could you merge these two definitions of the `StringsHelper`?

Please, add notes to clarify the final goal since this is a temporary solution.

Reply:
What I did is following the pattern from NetFx for future purposes when we have localization available in netcore. Here is what is happening in NetFx, the in path of 

1. `SqlClient\src\Microsoft.Data.SqlClient\netfx\tools` there is a script and a target file
2. The above files creates a new partial class named `StringHelpers` in `obj` folder and those are the files that gets localized versions as well.

My first goal was to add them in that folder without any localization and just English, but noticed this shortcut can do the job, running PowerShell is not supported in Unix and that prevents building the driver.  In future we can delete this class and move it `obj` folder as well. we can also have a `MetaDataTypeAttribute` class for annotations and have those created for Windows platform only. 

The entire design in this PR was to follow that pattern and avoid any unnecessary changes in future.